tyk-docs/data/menu.yaml

```
@@ -2098,6 +2094,30 @@ menu:
                 path: /product-stack/tyk-gateway/middleware/endpoint-plugin-tyk-classic
                 category: Page
                 show: True
+          - title: "Serving plugins"
```

```suggestion
          - title: "Plugin bundles"
```

This section is exclusively about plugin bundles, so again I would use the obvious name.